cmd/hubagent/workload/setup.go

```diff
@@ -183,26 +171,9 @@ func SetupControllers(ctx context.Context, wg *sync.WaitGroup, mgr ctrl.Manager,
 	}
 
 	rateLimiter := options.DefaultControllerRateLimiter(opts.RateLimiterOpts)
-	var clusterResourcePlacementControllerV1Alpha1 controller.Controller
 	var clusterResourcePlacementControllerV1Beta1 controller.Controller
 	var resourcePlacementController controller.Controller
 	var memberClusterPlacementController controller.Controller
-	if opts.EnableV1Alpha1APIs {
-		for _, gvk := range v1Alpha1RequiredGVKs {
-			if err = utils.CheckCRDInstalled(discoverClient, gvk); err != nil {
-				klog.ErrorS(err, "unable to find the required CRD", "GVK", gvk)
-				return err
-			}
-		}
-		klog.Info("Setting up clusterResourcePlacement v1alpha1 controller")
-		clusterResourcePlacementControllerV1Alpha1 = controller.NewController(crpControllerV1Alpha1Name, controller.NamespaceKeyFunc, pc.ReconcileV1Alpha1, rateLimiter)
-		klog.Info("Setting up member cluster change controller")
-		mcp := &memberclusterplacement.Reconciler{
```

Comment:
Can we keep this? As we are not removing code for this reconciler yet.

Reply:
sure